templates/include/glean/text.1.schema.json

```
@@ -0,0 +1,4 @@
+{
+  "type": "string",
+  "maxLength": 200000
```

Should this be 200kb?

```suggestion
  "maxLength": 204800
```